code/weapon/weapons.cpp

```diff
@@ -627,15 +627,26 @@ void parse_wi_flags(weapon_info *weaponp, flagset<Weapon::Info_Flags> preset_wi_
     if (!optional_string("$Flags:"))
         return;
 
+	// have to do this slightly out of order in order to handle spawn properly
+	// skip OVER flags, check for +override and reset num_spawn_weapons_defined
+	// otherwise if done afterward, new and old spawn weapons can get mixed up
+	pause_parse();
+	SCP_string flags;
+	stuff_string(flags, F_RAW);
```

Comment:
Shouldn't this use `stuff_string_list()` in case "+override" isn't on a separate line?

Reply:
I think so

Reply:
Done